doc/src/contribute/index.md

```diff
@@ -2,6 +2,15 @@
 title: Contributing to Sui
 ---
 
+## See our roadmap
+
+Sui is evolving quickly. See our [roadmap](https://github.com/MystenLabs/sui/blob/main/ROADMAP.md) for the
+overall status of Sui, including timelines for launching devnet, testnet, and mainnet.
+
+## Join the community
+
+To connect with the Sui community, join our [Discord](https://discord.com/invite/Gkh9xaWm).
```

Comment:
This is an invalid invite link. I believe the latest is `http://discord.gg/mysten`
![CleanShot 2022-03-30 at 00 01 32](https://user-images.githubusercontent.com/76067158/160771157-22c19e04-43d4-4978-8f13-9e9f669ee65c.png)
```suggestion
To connect with the Sui community, join our [Discord](http://discord.gg/mysten).
```